src/ProjectsElasticSearch.cpp

```diff
@@ -89,22 +89,22 @@ bool ProjectsElasticSearch::bulkRequest(eEvent& event) {
 }
 
 void ProjectsElasticSearch::addDoc(Int64 inodeId, std::string json) {
-  std::string url = getElasticSearchUpdateDocUrl(mIndex, inodeId);
+  std::string url = getElasticSearchUpdateDocUrl(mIndex, DEFAULT_TYPE, inodeId);
```

Comment:
same as before the type issue, it is not limited to ProjectsElasticSearch

Reply:
fixed